### ISSUE TITLE
Postal Code in final address

### ISSUE THREAD

**Issue by atwellpub** [callamwilliams/ACF-GoogleMaps-autocomplete#1]
Hi there! 

Great tool, thank you 🙌

We're noticing that before save, an address without a zip code is prompted for selection, but after save, an address with a zip code is populated. 

We'd like to remove the zip code from the final address. Is there a known way to do this?

Best regards,
H

**Comment by mt-deva** [comment]
Hi, I don't think so as that is default Autocomplete behaviour. You could do some workarounds to create a custom display input - but it'll feel messy so probably not worth the trade off.

**Comment by atwellpub** [comment]
I managed to figure out what was adding the zip code. It's this bit from `input.js`

https://github.com/callamwilliams/ACF-GoogleMaps-autocomplete/blob/497c6894b00f9a507b84f994c13ba84fa022b2ef/assets/js/input.js#L58

It seems when the `const place = autocomplete.getPlace();` is called in the `place_changed` hook, the formatted_address for the place object includes the zip. 

I was able to use this information to solve my problem. 

Gracias!
